app/index.jsx

```diff
@@ -1,3 +1,7 @@
+/* eslint-disable */
```

Comment:
I'd much prefer just disabling eslint for the two lines below. I don't want the entire file ignored for these two lines.